Bobsar0/Fast-Food-Fast
# #161570901 Manage menu items - implement food search

#### What does this PR do?
Implement food search functionality on the menu

#### Description of Task to be completed?
- Add script to display hide or display food items on the menu upon search

#### How should this be manually tested?
- Clone the repo and run the following commands:
```
  - git checkout ft-manage-menu-items-#161570901  
  - npm install
  - npm start
```
- Navigate to http://localhost:9999/ and login/create an account 

_You should be redirected to a dedicated menu page_
- Use the search feature at the top of the table to search for a food item.
- The food item(s) with relevant section should be displayed if available

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/#161570901 

#### Screenshots
![image](https://user-images.githubusercontent.com/19636985/48082624-630e8a80-e1f3-11e8-9368-3795a6b57546.png)
